readme.md

```diff
@@ -1,14 +1,95 @@
 # Inexor
 
-Inexor is a fork of the quake-like FPS Cube 2: Sauerbraten.
-It adds a lot of functionality and tries to stay open for improvements and suggestions.
-The task is to be more flexible and create an environment where developement is easy and fast and creativity can prosper.
+Inexor is a fork of the open-source First-Person-Shooter Cube 2: Sauerbraten, a fast-paced shooting game featuring an ingame map editor.  
+In contrast to Sauerbraten, Inexor adds a lot of functionality and strives to stay open to improvements and suggestions.  
+The goal of this project is to be more flexible and create an environment where development is easy, fast and where creativity can prosper.
 
-You will find more information in our wiki.
+## How is Inexor organized?
 
+We are a non-hierarchical organisation. This means we are simply a group of people with different ideas working together without a leader making all the decisions. Any one of us is free to work on the particular things he/she wants to.
+For this organisation to work properly we rely on good communication. We are on IRC and Mumble pretty much everyday. Every so often we organise official Mumble meetings to discuss our roadmap and strategies.
+
+To contribute to the project and merge your changes into the master branch you need to convince enough members that the changes are an improvement.
+If you keep communicating what you are doing, take into account feedback and tips from others as well as giving insights into your decision making process, it probably won't be an issue (as long as it doesn't collide with our roadmap, of course).
```

Comment:
That sounds a bit like there may likely be problems…

I think it suffices to say that we welcome contributions; that one needs to communicate well is somewhat a given IMHO.